components/dialog/index.css

```diff
@@ -11,377 +11,441 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Dialog {
-  /* Be a flexbox to allow a full sized content area that scrolls */
-  display: flex;
-
-  /* Allow 100% width, taking into account padding */
-  box-sizing: border-box;
-
-  /* Be no bigger than max-width,  but also be 90% if the viewport is smaller than max-width */
-  inline-size: fit-content;
-  min-inline-size: var(--mod-dialog-min-inline-size, var(--spectrum-dialog-min-inline-size));
-  max-inline-size: 100%;
-
-  max-block-size: inherit;
-
-  outline: none;
+	/* Be a flexbox to allow a full sized content area that scrolls */
+	display: flex;
+
+	/* Allow 100% width, taking into account padding */
+	box-sizing: border-box;
+
+	/* Be no bigger than max-width,  but also be 90% if the viewport is smaller than max-width */
+	inline-size: var(--mod-standard-dialog-max-width, var(--spectrum-standard-dialog-max-width));
+	min-inline-size: var(--mod-standard-dialog-min-inline-size, var(--spectrum-standard-dialog-min-inline-size));
+	max-inline-size: 100%;
+	max-block-size: inherit;
+	background-color: var(--spectrum-standard-dialog-background-color);
+	outline: none;
+	border-radius: var(--mod-standard-dialog-border-radius, var(--spectrum-standard-dialog-border-radius));
+	overflow: hidden;
+
+	.spectrum--large & {
+		--spectrum-standard-dialog-spacing-edge-to-content: var(--spectrum-spacing-400);
+	}
 }
 
 .spectrum-Dialog--small {
-  inline-size: var(--mod-dialog-confirm-small-width, var(--spectrum-dialog-confirm-small-width));
-}
-
-.spectrum-Dialog--medium {
-  inline-size: var(--mod-dialog-confirm-medium-width, var(--spectrum-dialog-confirm-medium-width));
+	--spectrum-standard-dialog-max-width: var(--spectrum-standard-dialog-maximum-width-small);
+	inline-size: var(--mod-standard-dialog-max-width-small, var(--spectrum-standard-dialog-max-width));
 }
 
 .spectrum-Dialog--large {
-  inline-size: var(--mod-dialog-confirm-large-width, var(--spectrum-dialog-confirm-large-width));
+	--spectrum-standard-dialog-max-width: var(--spectrum-standard-dialog-maximum-width-large);
+	inline-size: var(--mod-standard-dialog-max-width-large, var(--spectrum-standard-dialog-max-width));
 }
 
 .spectrum-Dialog-hero {
-  grid-area: hero;
-  block-size: var(--mod-dialog-confirm-hero-height, var(--spectrum-dialog-confirm-hero-height));
+	grid-area: hero;
 
-  border-start-start-radius: var(--mod-dialog-confirm-border-radius, var(--spectrum-dialog-confirm-border-radius));
-  border-start-end-radius: var(--mod-dialog-confirm-border-radius, var(--spectrum-dialog-confirm-border-radius));
+	block-size: var(--mod-standard-dialog-hero-block-size, var(--spectrum-standard-dialog-hero-block-size));
 
-  background-size: cover;
-  background-position: center center;
-  overflow: hidden;
+	/* border-start-start-radius: var(--mod-standard-dialog-border-radius, var(--spectrum-standard-dialog-border-radius));
+	border-start-end-radius: var(--mod-standard-dialog-border-radius, var(--spectrum-standard-dialog-border-radius)); */
```

Comment:
Nope! I kept them as we were deciding some of the[ WHCM mode stuff](https://github.com/adobe/spectrum-css/pull/2860#discussion_r1666967533), just in case I had to revert or something.